Dockerfile

```diff
@@ -1,8 +1,35 @@
-FROM alpine:3.7
+FROM alpine:3.7 as base
+# Install go binary for make bin/gcsproxy
+COPY --from=golang:1.13-alpine /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
 ENV GCSPROXY_VERSION=0.3.0
-RUN apk add --update ca-certificates
-RUN apk add --no-cache --virtual .build-deps ca-certificates wget \
-  && wget https://github.com/daichirata/gcsproxy/releases/download/v${GCSPROXY_VERSION}/gcsproxy_${GCSPROXY_VERSION}_amd64_linux -O /usr/local/bin/gcsproxy \
+
+FROM base as release
+WORKDIR /app
+COPY . .
+RUN apk add --no-cache make \
+  && make bin/gcsproxy \
+  && cp bin/gcsproxy /usr/local/bin/gcsproxy \
   && chmod +x /usr/local/bin/gcsproxy
 
-CMD ["gcsproxy"]
+FROM base as devel_base
```

Comment:
Can we revert the `Dockerfile` changes here - GDB is not a very common option here.

See 
- https://golang.org/doc/gdb
-https://github.com/derekparker/delve
- https://medium.com/@kaperys/delve-into-docker-d6c92be2f823

In general, this seems more like a debugging issue (workaround) than a general devel toolkit that should exist in a `Dockerfile`.